classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def student(param)
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the name of the method is descriptive, you can rename this method to `add_student` instead